### ISSUE TITLE
Screen reader is not picking up a11y hints for any of the pages

### ISSUE THREAD

**Issue by esther-k** [department-of-veterans-affairs/va.gov-team#15246]
## What Happened?
While using a screen reader, the a11y hints for elements on all of the current pages are not being read. 

## Specs:
- Android Pixel 3 emulator
- With TalkBack enabled
- Any user

## Steps to Reproduce
- Enable TalkBack and select an element (i.e. button) on any page

## Desired Behavior
- When an element is selected, the screen reader should be reading out the a11y hint lined out in the implementation guideline for that element. 

## Acceptance Criteria
- [ ] The screen reader will read out the a11y hints for all elements 

## Bug Severity
- 3 - Medium

## Screen Shot (s) and Additional Information
The screen reader does not pick the a11y hints for the highlighted areas. 

_Note: All of the elements for the profile & contact info page are not included in the images but they **are** affected._

![Screen Reader a11y Hint - Home.png](https://images.zenhubusercontent.com/5f7bb216620a259ab7e50d60/cb5d7233-fb79-4e11-8fb2-f19cb716150f)
![Screen Reader a11y Hint - Crisis Line.png](https://images.zenhubusercontent.com/5f7bb216620a259ab7e50d60/95b2065c-a743-4914-b3a4-4a9ab3ccccdc)
![Screen Reader a11y Hint - Contact VA.png](https://images.zenhubusercontent.com/5f7bb216620a259ab7e50d60/f92aa80a-7819-486d-8148-d6fac1ec9a4f)
![Screen Reder a11y Hint - Profile Info.png](https://images.zenhubusercontent.com/5f7bb216620a259ab7e50d60/1593c67c-7f89-4507-8fa1-49715207678c)
![Screen Reader - Direct Deposit.png](https://images.zenhubusercontent.com/5f7bb216620a259ab7e50d60/1e7a7c97-0be7-4fbb-9fac-8f6cbdf0fe21)

**Comment by esther-k** [comment]
**11/24**

PASS - on iOS and android. Outside of a few isolated cases, a11yHints are being picked up the screen reader
